azure-pipelines.yml

```diff
@@ -123,6 +123,7 @@ jobs:
         PYTHON_VERSION: '*'
         BLAS: 'mkl'
         COVERAGE: 'true'
+        SHOW_SHORT_SUMMARY: 'true'
```

Comment:
It is weird because it seems that the short summary is not shown only for this build while the env variable indicates the opposite?

Reply:
This was fixed in subsequent commits apparently.